seaworthy/pytest/fixtures.py

```diff
@@ -35,9 +35,7 @@ def fixture():
     return fixture
 
 
-"""
-Default fixture for the ``DockerHelper``. Has module scope.
-"""
+#: Default fixture for the :class:`.DockerHelper`. Has module scope.
```

Comment:
`#:` 🤔 learn something new every day.

Reply:
You can also write the docstring *after* the attribute being documented, but I find that more confusing.